Makefile

```diff
@@ -83,12 +96,6 @@ verify: ## Verify code. Includes codegen, docgen, dependencies, linting, formatt
 	go vet ./...
 	cd kwok/charts && helm-docs
 	golangci-lint run
-	@git diff --quiet ||\
```

Comment:
Removed this since the GHA that I was using thought there were changes in the LICENSE and README files. Will add back in once i've figured out what the linter was doing differently here.